ui/src/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestStatistics.js

```diff
@@ -43,13 +43,13 @@ export class BorrowingRequestStatistics extends React.Component {
     }
   }
 
-  renderLoanEnds() {
-    const { loan_end_date: loanEndDate } = this.props.brwReq;
+  renderDueDate() {
+    const { due_date: dueDate } = this.props.brwReq;
```

Comment:
Maybe we can put the props that are used in the component like  `due_date` and the `status` in the propTypes so we start becoming more explicit, lets not collide though with @kprzerwa

Reply:
This requires quite a huge change. I am not sure we want to have the object shape of an entire record honestly. If yes, then IMHO we will have to create strict backend serializers. To be discussed.